configs/webpack.docs.dev.config.js

```diff
@@ -177,6 +177,56 @@ module.exports = {
             to: path.join(process.cwd(), 'assets', 'css')
         }]),
 
+        new CopyWebpackPlugin([{
```

Comment:
Simplify this group using an array of objects rather than separate calls each time, eg:

![image](https://cloud.githubusercontent.com/assets/20795331/24998175/261a498c-2031-11e7-9f7c-41413da7f1eb.png)